llvm/test/tools/sycl-post-link/spec-constants/spec_const_and_split.ll

```diff
@@ -2,27 +2,48 @@
 ; device code splitting and specialization constant processing are
 ; requested.
 ;
+; Additionally this test checks that sycl-post-link creates specialization
```

Comment:
Maybe also add check for a kernel without any spec consts?

Reply:
This might be a good idea, added in e25a122570b1a89c7ee188e4b6fdbaa8c2c0bc8c